simple/src/main/g8/build.sbt

```diff
@@ -0,0 +1,45 @@
+$!
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+!$
+name := "dfdl-$name;format="norm"$"
+
+organization := "$package$"
+
+version := "0.1.0-SNAPSHOT"
+
+scalaVersion := "2.12.11"
+
+libraryDependencies ++= Seq(
+  "org.apache.daffodil" %% "daffodil-tdml-processor" % "$daffodil_version$" % "test",
+  "com.novocode" % "junit-interface" % "0.11" % "test",
+  "junit" % "junit" % "4.12" % "test",
```

Comment:
Should we update junit dependencies too?  I also found out that adding log4j was needed to avoid a warning too.

```
  "com.github.sbt" % "junit-interface" % "0.13.2" % "test",
  "junit" % "junit" % "4.13.2" % "test",
  "org.apache.daffodil" %% "daffodil-tdml-processor" % "$daffodil_version$" % "test",
  "org.apache.logging.log4j" % "log4j-core" % "2.17.1" % "test",
```